envi/tests/test_arch_i386_emu.py

```diff
@@ -220,12 +220,37 @@ def test_i386_reps(self):
         emu32.setRegisterByName('ecx', -1)
         emu32.setRegisterByName('edi', 0x40000000)
         emu32.executeOpcode(op_scasb)
-        self.assertEqual(emu32.getRegisterByName('ecx'), 0xfffffff1)
+        self.assertEqual(emu32.getRegisterByName('ecx'), 0xffffffff - len("foobarbazbane\x00"))
```

Comment:
just making this easier to understand for a human

Reply:
:+1: